index.html

```diff
@@ -0,0 +1,226 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Playfair+Display:ital,wght@0,400;0,500;0,600;0,700;0,800;0,900;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Playfair+Display:ital,wght@0,400;0,500;0,600;0,700;0,800;0,900;1,400;1,500;1,600;1,700;1,800;1,900&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header class="header-file">
+        <img src="images/image-1.svg" alt="">
+        <nav class="header">
+        <input class="menu-btn" type="checkbox" id="menu-btn" />
+        <label class="menu-icon" for="menu-btn"><span class="nav-icon"></span></label>
+        <ul class="menu">
+            <li><a href="#dont">Hello</a></li>
+            <li><a href="#forget">Project</a></li>
+        </ul>
+        </nav>
+    </header>
+    <section class="section">
+        <div class="info-about-me">
+            <h1 class="my-name">Abdulai <br> Samuel <br> Moiwa</h1>
+            <p class="info">Hello! I am a software developer! I can <br>
+                help you build a product, feature or <br>
+                website. Taka a look of my works. <br>
+                If you like what you see and have a <br>
+                project you need coded, dont hesitate <br>
+                and contact me.</p>
+        </div>
+        <button class="collaboration" type="button">Start collaboration</button>  
+        <img class="banner-image" src="images/image-2.svg" alt=""> 
```

Comment:
- [ ] I think you should consider adding this image as a background because it is not part of the content, it is just for design/decoration purposes. Kindly check out this [article ](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more information 📱